README.md

```diff
@@ -209,10 +214,31 @@ services:
         group-id: <gid>
 ```
 
-## TODO/Contributing
+## API and clients
+
+The Pebble daemon exposes an API (HTTP over a Unix socket) to allow remote clients to interact with the daemon. It can start and stop services, add configuration layers the plan, and so on. There is currently no official documentation for the API (apart from the [code itself](https://github.com/canonical/pebble/blob/master/internal/daemon/api.go)!); most users will interact with it via the Pebble command line interface or the Go or Python client.
+
+The [Go client](https://pkg.go.dev/github.com/canonical/pebble@v0.0.0-20210907234803-4c5d8c174b10/client) is used by the CLI to connect to the Pebble API. You can use this as follows:
```

Comment:
Prefer to link to version agnostic links like https://pkg.go.dev/github.com/canonical/pebble/client

Reply:
Good call, done. It's a bit annoying that when you click on Directories > client in the pkg.go.dev interface it gives you the version-specific link.